src/Symfony/Component/Form/Console/Descriptor/TextDescriptor.php

```diff
@@ -183,20 +183,16 @@ private function normalizeAndSortOptionsColumns(array $options)
 
     private function getDumpFunction()
     {
-        $cloner = new VarCloner();
-        $cloner->addCasters(['Closure' => function ($c, $a) {
-            $prefix = Caster::PREFIX_VIRTUAL;
+        if (class_exists(Dumper::class)) {
+            return new Dumper($this->output);
+        }
 
-            return [
-                $prefix.'file' => $a[$prefix.'file'],
-                $prefix.'line' => $a[$prefix.'line'],
-            ];
-        }]);
-        $dumper = new CliDumper(null, null, CliDumper::DUMP_LIGHT_ARRAY | CliDumper::DUMP_COMMA_SEPARATOR);
-        $dumper->setColors($this->output->isDecorated());
+        // BC symfony/console <4.3
```

Comment:
it's not BC at this point, because nothing has been deprecated

Reply:
what about "preserve compatibility with symfony/console <4.3."?

Reply:
what about bumping by adding a conflict symfony/console <4.3 instead?

Reply:
Actually that wouldn't change much the code here :)

Reply:
I would just remove the comment :)

Reply:
wasnt sure about the conflict rule :) now fixed.

Reply:
and missed your comments ..., i think always using the Console dumper is the ideal scenario (and now doing the same in #27684 )